build.gradle

```diff
@@ -125,13 +126,34 @@ jacocoTestReport {
   }
 }
 
+flyway {
+  url = System.getenv('POSTGRES_HOST') ?: "localhost"
```

Comment:
Is it better to use `127.0.0.1` as the default so it uses TCP/IP instead of the socket, for when running Postgres in Docker?